core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -215,6 +215,12 @@ oppia.controller('CollectionPlayer', [
       return completedExplorationIds.indexOf(explorationId) > -1;
     };
 
+    $scope.isNextExploration = function(explorationId) {
+      var nextExplorationId = (
+        $scope.collectionPlaythrough.getNextExplorationIds()[0]);
```

Comment:
yep, I tested manually and looks like it works. Actually, when I checked getNextExplorationIds() just returns a single valued list.